drivers/i2c/i2c_tca954x.c

```diff
@@ -186,6 +191,7 @@ BUILD_ASSERT(CONFIG_I2C_TCA954X_CHANNEL_INIT_PRIO > CONFIG_I2C_TCA954X_ROOT_INIT
 		.nchans = ch,							  \
 		.reset_gpios = GPIO_DT_SPEC_GET_OR(			          \
 				DT_INST(inst, ti_tca##n##a), reset_gpios, {0}),	  \
+		.idle_disconnect = DT_INST_PROP_OR(inst, i2c-mux-idle-disconnect, 0),	\
```

Comment:
```suggestion
		.idle_disconnect = DT_INST_PROP_OR(inst, i2c_mux_idle_disconnect, 0),	\
```
According to `DT_INST_PROP_OR` documentation, the property name should follow the lowercase-and-**underscores** convention.
<img width="413" alt="image" src="https://github.com/user-attachments/assets/3a1930ef-ae4d-4d55-9069-666136135528" />